incubator/riemann/values.yaml

```diff
@@ -0,0 +1,44 @@
+# replicaCount is expected to be 1 right now. Riemann
+# does not support horizontal scaling out of the box.
+replicaCount: 1
+image:
+  repository: raykrueger/riemann:0.2.14
```

Comment:
Separate repo and tag.

```yaml
image:
  repository: raykrueger/riemann
  tag: 0.2.14
```

Reply:
I wasn't seeing a lot of consistency here among the stable images, so I went with the simplest form. Is there tooling that expects separate values?